recipes/libtommath/cmake/conandata.yml

```diff
@@ -0,0 +1,11 @@
+sources:
+  # Newer versions at the top
+  "1.2.1-dev":
```

Comment:
We typically do not allow unofficial releases

https://github.com/libtom/libtommath/releases given its been almost 4 years, this should be done using out "cci specific" format

https://github.com/conan-io/conan-center-index/blob/c66c44e5be27c7cb230c8464638b82b1f504bb60/docs/adding_packages/conanfile_attributes.md?plain=1#L43

Reply:
Thank you for pointing me into the right direction. I have adopted the cci-format.